src/m3nsch/examples/sample_workload.go

```diff
@@ -1,3 +1,5 @@
+//+build never
```

Comment:
lol why?

Reply:
It was causing lint errors or some similar build error when I was testing and I wanted to ignore it, I'll loop back and try to figure out what that was